src/foundation-player.coffee

```diff
@@ -202,10 +205,8 @@
     # Look and feel ============================================================
     # This method toggles player size
     togglePlayerSize: ->
-      switchToSize = if @currentPlayerSize == 'normal' then 'small' else 'normal'
-      switchClass @$wrapper, switchToSize, @currentPlayerSize
-      @setPlayerSizeHandler()
-      @currentPlayerSize = switchToSize
+      setPlayerSize('small') if @currentPlayerSize == 'normal'
+      setPlayerSize('normal') if @currentPlayerSize == 'small'
```

Comment:
You reading default value of `currentPlayerSize` here `setPlayerSize('normal') if @currentPlayerSize == 'small'`.
After switching size `currentPlayerSize` doesn't get updated. Right? Have you tested `currentPlayerSize` attribute value?